js/.eslintrc

```diff
@@ -169,7 +169,7 @@
         "prefer-const": 0,
         "prefer-reflect": 0,
         "prefer-spread": 2,
-        "prefer-template": 2,
+        "prefer-template": 0, // not replace content in string
```

Comment:
I think we can use this in js files : ``/*eslint prefer-template: 0*/`` so it would be ok 
What do you think about this ?